src/validation.cpp

```diff
@@ -4079,6 +4085,7 @@ bool CChainState::ReplayBlocks()
     int nForkHeight = pindexFork ? pindexFork->nHeight : 0;
     for (int nHeight = nForkHeight + 1; nHeight <= pindexNew->nHeight; ++nHeight) {
         const CBlockIndex* pindex = pindexNew->GetAncestor(nHeight);
+        assert(pindex != nullptr);
```

Comment:
```suggestion
        const CBlockIndex& block{*Assert(pindexNew->GetAncestor(nHeight))};
```

If the value is used later, you may as well convert it to a reference after the assert.

Reply:
Interesting, I added it